packages/expect/src/jest-utils.ts

```diff
@@ -135,10 +135,6 @@ function eq(
   if (typeof a !== 'object' || typeof b !== 'object')
     return false
 
-  // Use DOM3 method isEqualNode (IE>=9)
-  if (isDomNode(a) && isDomNode(b))
```

Comment:
We can't just remove this because someone wants to.

Reply:
@sheremet-va I agree, but wouldn't this be something that should enhance functionality? e.g. if you're using the JSDOM adapter? if you're not writing tests for non-browser APIs, why should it exist?

Reply:
There is no JSDOM adapter for `equality`. If you are afraid of the Proxy usage here we can change it in a few other ways:

- Check that global `Element` exists and check `instanceof` on the value.
- Instead of direct access `obj.nodeType` use `'nodeType' in obj && obj.nodeType`

Reply:
@sheremet-va that's a great suggestion :) Updated.